### PR TITLE
Companion Substaret/5683

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -308,6 +308,7 @@ parameter_types! {
 	pub const MaxNominatorRewardedPerValidator: u32 = 64;
 	// quarter of the last session will be for election.
 	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
+	pub const MaxIterations: u32 = 5;
```
Comment (bkchr) [inline runtime/kusama/src/lib.rs:311]: Why only 5 here?

Comment (bkchr) [reply runtime/kusama/src/lib.rs:311]: And a better name `StakingMaxIterations` or whatever would also be useful.

Comment (kianenigma) [reply runtime/kusama/src/lib.rs:311]: For Kusama and Polkadot I set to 5 to be more cautious not to stall solution submission for too long. If all is well we can increase it in a further upgrade.

Comment (bkchr) [reply runtime/kusama/src/lib.rs:311]: Okay

**Pull request merged**